wdl/workflows/minimap.wdl

```diff
@@ -19,6 +19,7 @@ workflow minimapWorkflow {
     }
 
 	output {
-        File minimapBam = minimap2.bam
+        File minimapBam = minimap2.bam,
+	File minimapBamBai = minimap2.bam.bai,
```

Comment:
The file variable name needs to be the name in the task WDL, which is `bamIndex`. I made the appropriate change in the other branch - which we will be merging in to this R9 branch. Sorry for the confusion but the fix is made.